### ISSUE TITLE
tag minor release?

### ISSUE THREAD

**Issue by musm** [glfw/glfw#1379]

**Comment by musm** [comment]
According to https://github.com/glfw/glfw/compare/3.2.1...master there have been roughly 600 commits since the previous release. I was wondering if it would be appropriate to tag a new release

**Comment by GreenLightning** [comment]
I agree that the last official release was quite a while back, but it doesn't make sense to just tag the latest commit as a new release, because features need to be complete, the release must be tested, the documentation must be updated, etc. The next release will be GLFW 3.3 and you can see the Milestone [here](https://github.com/glfw/glfw/milestone/12), which is currently at 96% completion. Once it has been officially released, the commit will be tagged accordingly.
